src/databases/WavefrontOBJ/avtWavefrontOBJOptions.C

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+// ************************************************************************* //
+//                             avtWavefrontOBJOptions.C                              //
+// ************************************************************************* //
+
+#include <avtWavefrontOBJOptions.h>
+
+#include <DBOptionsAttributes.h>
+
+#include <string>
+
+
+// ****************************************************************************
+//  Function: GetWavefrontOBJReadOptions
+//
+//  Purpose:
+//      Creates the options for WavefrontOBJ readers.
+//
+//  Important Note:
+//      The code below sets up empty options.  If your format 
+//      does not require read options, no modifications are 
+//      necessary.
+//
+//  Programmer: Justin Privitera
+//  Creation:   11/03/23
+//
+// ****************************************************************************
+
+DBOptionsAttributes *
+GetWavefrontOBJReadOptions(void)
+{
+    DBOptionsAttributes *rv = new DBOptionsAttributes;
+    return rv;
+    
+/* EXAMPLE OF OPTIONS
```

Comment:
Do we need this boilerplate autogen'd stuff here? I don't think we do.

Reply:
I think I've seen that in other plugins (https://github.com/visit-dav/visit/blob/develop/src/databases/Blueprint/avtBlueprintOptions.C#L45). But it is not included in all plugins. Should I remove it here?

Reply:
Removed.